cli/chaosli/cmd/root.go

```diff
@@ -101,6 +98,16 @@ func initConfig() {
 	}
 }
 
+func initDDMark() {
+	// setup ddmark
+	var err error
+
+	ddMarkClient, err = ddmark.NewClient(v1beta1.EmbeddedChaosAPI)
+	if err != nil {
+		log.Fatal("ddmark didn't init properly")
```

Comment:
nit: Is it worse to add the error message to the log? _I think it could help to understand the error._